model/model_training/trainer_sft.py

```diff
@@ -252,6 +252,9 @@ def argument_parsing(notebook=False, notebook_args=None):
         gradient_accumulation_steps=training_conf.gradient_accumulation_steps,
         per_device_train_batch_size=training_conf.per_device_train_batch_size,
         per_device_eval_batch_size=training_conf.per_device_eval_batch_size,
+        adam_beta1=training_conf.adam_beta1,
+        adam_beta2=training_conf.adam_beta2,
+        adam_epsilon=training_conf.adam_epsilon,
```

Comment:
float(training_conf.adam_epsilon)

Reply:
Is this cast a suggested change? Only for adam_epsilon or for beta1/beta2 also.

Reply:
yaml loads floats in scientific notation as strings unless you use a dot in the mantissa. So either we change the yaml or add a cast here